docs/regmap/adi_regmap_dmac.txt

```diff
@@ -368,7 +368,7 @@ X_LENGTH
 ENDREG
 
 FIELD
-[23:0] 2^log2(max(''DMA_DATA_WIDTH_SRC'', ''DMA_DATA_WIDTH_DEST'')/8)-1
+[31:0] 2^log2(max(''DMA_DATA_WIDTH_SRC'', ''DMA_DATA_WIDTH_DEST'')/8)-1
```

Comment:
I agree to have the length as the maximum possible value of DMA_LENGTH_WIDTH instead of the default.